src/v2/Apps/Artist/Components/ArtistCollectionsRail/__tests__/ArtistCollectionsRail.jest.tsx

```diff
@@ -21,14 +23,11 @@ describe("CollectionsRail", () => {
     props = {
       collections: CollectionsRailFixture,
     }
+    ;(paginateCarousel as jest.Mock).mockImplementation(() => [0, 100, 200])
```

Comment:
Out of curiosity what does "100, 200" mean?

Reply:
Ah, it could literally be any numeric array — they are page stops. So this would be a carousel with 3 pages at those offsets.